float64 pose_x
float64 pose_y
float64 pose_z
float64 orientation_x
float64 orientation_y
float64 orientation_z
float64 orientation_w
---
int64 error_code
